garage/envs/multitask_env.py

```diff
@@ -0,0 +1,67 @@
+"""Multitask environment."""
+
+import gym
+import numpy as np
+
+from garage.core import Serializable
+
+
+class MultitaskEnv(gym.Wrapper, Serializable):
+    """
+    Multitask Environment wrapper.
+
+    This class take an task-based environment and wrap it.
+    It provides functions like resetting/sampling tasks
+    that is used for multitask sampling.
+    """
+
+    def __init__(self, wrapped_env, tasks, task_sample_method="round_robin"):
```

Comment:
What is the data type of `tasks`? If `wrapped_env` needs a task space, then what is the purpose of tasks?